subprojects/core/src/integTest/groovy/org/gradle/groovy/scripts/GradleUtilDefaultImportDeprecationIntegrationTest.groovy

```diff
@@ -22,6 +22,10 @@ import spock.lang.Issue
 @Issue('https://github.com/gradle/gradle/issues/1793')
 class GradleUtilDefaultImportDeprecationIntegrationTest extends AbstractIntegrationSpec {
 
+    def setup() {
+        executer.requireOwnGradleUserHomeDir()
```

Comment:
Why is this required?

Reply:
The test relies on an empty compiled script cache. If the script is cached, it won't be compiled and there will be no warning.

This is working on CI because we do clean builds, but it fails locally if you execute the test more than once.

Reply:
Good catch!